ThunkLibs/Generator/main.cpp

```diff
@@ -42,6 +43,10 @@ int main(int argc, char* const argv[]) {
   const std::string libname = *arg++;
   const std::string target_abi = *arg++;
   const std::string output_filename = *arg++;
+  std::optional<std::string> x86_rootfs = std::string {*arg++} + "/usr/";
+  if (x86_rootfs.value() == "//usr/") {
```

Comment:
This sequence seems convoluted, but I don't know enough C++ to offer a better solution.

Reply:
Ah yes, good point.

Is the following any better?
```cpp
  std::optional<std::string> x86_rootfs;
  if (arg != std::string_view { "/" }) {
    x86_rootfs = std::string { *arg } + "/usr/";
  }
  ++arg;
```

Reply:
Luckily none of this is actually needed anymore now that `--sysroot` is used instead.

Reply:
Yes that's about what I'd expect :)